src/targets/r/httr/client.ts

```diff
@@ -32,24 +32,22 @@ export const httr: Client = {
 
     // Construct query string
     const qs = queryObj;
-    const queryCount = Object.keys(qs).length;
     delete queryObj.key;
 
-    if (queryString.length === 1) {
+    const queryCount = Object.keys(qs).length;
+    if (queryCount === 1) {
       push(`queryString <- list(${Object.keys(qs)} = "${Object.values(qs).toString()}")`);
       blank();
-    } else if (queryString.length > 1) {
-      let count = 1;
-
+    } else if (queryCount > 1) {
       push('queryString <- list(');
 
-      for (const query in qs) {
-        if (count++ !== queryCount - 1) {
+      Object.keys(qs).forEach((query, i) => {
```

Comment:
love it! if we could do away with every single for loop, I'd be in favor!

the only thing I'll mention is just a reminder that in JavaScript objects are unordered, so the sort order here is technically undefined (in Safari's WebKit engine, for example, it would be different than Chrome's V8).